src/index.html

```diff
@@ -40,6 +41,26 @@ <h1 class="title1">Travel with Friends</h1>
 			starting from
 			ROSE KENNEDY GREENWAY</p>
 		</article>
+
+		<article class="box2">
+			<div class="minibox2">
+					<form action= "">
+						<fieldset>
+							<input type = "checkbox"
+							class = "chkspot2"
+							value = "rkg" />
+							<label for = "chkspot2">The Daily Catch</label>
```

Comment:
You're getting the error above because you have extra spacing in around your `for` here - remove that and the label should connect up.